src/main/java/com/logentries/net/LogentriesClient.java

```diff
@@ -227,23 +227,8 @@ private SSLContext CreateAndInitSSLContext(String algorithm) throws NoSuchAlgori
 			KeyManagementException {
 
 		SSLContext sslContext = SSLContext.getInstance(algorithm);

```

Comment:
we could - but as you can see the code is written to allow creation of an instance of the context with a choice of protocol - why? I have no idea, but I'm sure it seemed like a good idea to however wrote it. I am deliberately trying to minimise changes to this library - would it be a big deal to leave it as is?

http://developer.android.com/reference/javax/net/ssl/SSLContext.html#getInstance%28java.lang.String%29